benchmark/wildtime_benchmarks/data_generation_arxiv.py

```diff
@@ -44,39 +48,89 @@ def __init__(self,  data_dir):
         self.path = data_dir
 
     def store_data(self, store_all_data: bool, add_final_dummy_year: bool):
-        for year in tqdm(self._dataset):
+        # create directories
+        if not os.path.exists(self.path):
+            os.mkdir(self.path)
+
+        train_dir = os.path.join(self.path, "train")
+        os.makedirs(train_dir, exist_ok=True)
```

Comment:
As a general note, I prefer `pathlib.Path` for such operations (esp. joining paths), it reads a bit more idiomatic. But since this is just data generation code, we can keep os.